### PR TITLE
AUTHORS,.mailmap: update with recent contributors

### PR DESCRIPTION
**Pull request by neersighted**
**- What I did**

Regenerate `AUTHORS` and update `.mailmap`, using GitHub profile data when it appeared higher quality than Git metadata.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -177,6 +177,8 @@ David M. Karr <davidmichaelkarr@gmail.com>
 David Sheets <dsheets@docker.com> <sheets@alum.mit.edu>
 David Sissitka <me@dsissitka.com>
 David Williamson <david.williamson@docker.com> <davidwilliamson@users.noreply.github.com>
+David Dooling <dooling@gmail.com>
+David Dooling <dooling@gmail.com> <david.dooling@docker.com>
```
Comment (thaJeztah) [inline .mailmap:181]: Can you sort this list? Looks like it's in the wrong position (should be before `David M.` probably)

File: AUTHORS [hunk 2]
```diff
@@ -2253,6 +2261,7 @@ VladimirAus <v_roudakov@yahoo.com>
 Vladislav Kolesnikov <vkolesnikov@beget.ru>
 Vlastimil Zeman <vlastimil.zeman@diffblue.com>
 Vojtech Vitek (V-Teq) <vvitek@redhat.com>
+voloder <110066198+voloder@users.noreply.github.com>
```
Comment (thaJeztah) [inline AUTHORS:2264]: 😢 Looks like this one went unnoticed (handle, not name)

File: AUTHORS [hunk 3]
```diff
@@ -731,6 +732,7 @@ Feroz Salam <feroz.salam@sourcegraph.com>
 Ferran Rodenas <frodenas@gmail.com>
 Filipe Brandenburger <filbranden@google.com>
 Filipe Oliveira <contato@fmoliveira.com.br>
+Filipe Pina <636320+fopina@users.noreply.github.com>
```
Comment (thaJeztah) [inline AUTHORS:735]: Looks like this one can use an entry in `.mailmap` for the e-mail address;

```bash
git log --author=636320+fopina@users.noreply.github.com
commit ef681124ca61f52e72d1e1c4fc589373d2a4e08e
Author: Filipe Pina <636320+fopina@users.noreply.github.com>
Date:   Thu Feb 29 23:23:19 2024 +0000

    fix typo in error message

    Signed-off-by: Filipe Pina <hzlu1ot0@duck.com>
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (thaJeztah) [reply .mailmap:181]: Oh! Looks like the most recent commits from David use his `@docker.com` address in the sign-off, so this should probably be the reverse;

```
David Dooling <david.dooling@docker.com>
David Dooling <david.dooling@docker.com> <dooling@gmail.com>
```

File: AUTHORS (hunk 2 above)
Comment (neersighted) [reply AUTHORS:2264]: This is intentional; the user does not have 'better' data on their GitHub profile.

File: .mailmap (hunk 1 above)
Comment (neersighted) [reply .mailmap:181]: I assumed here that @whalelines preferred the original email here; can you confirm?

File: AUTHORS (hunk 3 above)
Comment (neersighted) [reply AUTHORS:735]: Good catch; missed this one!

File: AUTHORS (hunk 2 above)
Comment (thaJeztah) [reply AUTHORS:2264]: Yeah, we should've ask them to provide when merging the PR.

**Review by whalelines — APPROVED** [review]
LGTM

File: AUTHORS (hunk 2 above)
Comment (neersighted) [reply AUTHORS:2264]: I don't see why -- if a user wants to be fully pseudonymous, that's their right. Plenty of other examples of that in this file, too.

Comment (thaJeztah) [reply AUTHORS:2264]: https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work

> Use your real name (sorry, no pseudonyms or anonymous contributions.)

Comment (neersighted) [reply AUTHORS:2264]: Ah, well, fair enough... That ship has sailed, I guess.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Failure is unrelated; flaky test;

```
=== FAIL: amd64.integration.plugin.logging TestReadPluginNoRead/explicitly_enabled_caching (1.29s)
    read_test.go:90: assertion failed: strings.TrimSpace(buf.String()) is not "hello world": []
    --- FAIL: TestReadPluginNoRead/explicitly_enabled_caching (1.29s)

=== FAIL: amd64.integration.plugin.logging TestReadPluginNoRead (5.11s)
    read_test.go:93: [d4da968388ed9] daemon is not started
```

**Pull request merged**